### DISCUSSION TITLE
What is the state of issue 6039?

### DISCUSSION THREAD

**Discussion by DerEwige** [lightningnetwork/lnd#7131] [General]
https://github.com/lightningnetwork/lnd/issues/6039

In the last months alone, I had about a dozen force closes cause by this wrong behavior of lnd.
The issuehas been opened nearly a year ago

afaik a short fix would be to send a warning instead of an error.
This should stop eclair and CLN from force closing

**Comment by yyforyongyu** [comment]
#6760 fixes it.

**Reply by DerEwige** [reply]
So this is currently sheduled to be included in 0.16.0 right?
Any when this release should be expected?
